core/src/main/java/org/commonjava/maven/ext/core/ManipulationManager.java

```diff
@@ -191,6 +192,10 @@ public void init( final ManipulationSession session )
     public void scanAndApply( final ManipulationSession session )
                     throws ManipulationException
     {
+        final PreparseGroovyManipulator preparseGroovyManipulator = new PreparseGroovyManipulator( null, null, pomIO );
+        preparseGroovyManipulator.init( session );
+        preparseGroovyManipulator.applyChanges( Collections.emptyList() );
```

Comment:
Given this manipulator **isn't** following the standard pattern (else as we discussed it would be accidentally run during the standard execution), what about making a simpler API ; i.e. merging `init` and `apply` methods?

Reply:
Sure.